confd/vars/main.yaml

```diff
@@ -125,5 +128,7 @@ kilda_gui_application_name_prefix: "Dev"
 
 kilda_grpc_endpoint: "http://grpc-speaker.pendev"
 kilda_grpc_rest_port: "8091"
+kilda_grpc_basicauth_username: "kilda"
+kilda_grpc_basicauth_password: "kilda"
 kilda_grpc_username: "kilda"
```

Comment:
It's better to specify the scope / the purpose of  "kilda_grpc_username" credentials.